tools/add_to.hpp

```diff
@@ -163,7 +165,7 @@ namespace antler {
          dep_subcommand->add_option("-t, tag", tag, "Tag associated with the dependency.");
          dep_subcommand->add_option("-r, release", release, "Release version of the depedency.");
          dep_subcommand->add_option("--digest, hash", hash, "Hash of the dependency.");
-         
+

```

Comment:
general suggestion, please avoid whitespace only changes, it just messing history and makes changes more verbose. Not necessary to fix it for now as it is already there, so just for future.

Reply:
Sorry, I didn't actually notice I'd done it until AFTER Mikal had reviewed. Auto-updated in my editor. I generally agree, white space changes are to be avoided.